docs/resources.html

```diff
@@ -146,6 +147,16 @@ <h4 id="pre-publication-verification-of-analyses">Pre-publication verification o
 				<p>Some journals have now adopted a policy whereby data and code are not only required for publication in the journal, but must be checked before publication to ensure that the analyses are reproducible – that the results in the manuscript match the results that are produced when someone who is not one of the authors re-runs the code on the data. This is called pre-publication “verification of analyses”, “data and code replication”, or “reproduction of analyses”. See <a href="https://doi.org/10.1162/99608f92.25982dcf" target="_blank">Willis & Stodden (2020)</a> for a useful overview of how to leverage policies, workflows, and infrastructure to ensure computational reproducibility in publication.</p>
 				<p>For more information on how to implement a policy like this at your journal, see the <a href="https://social-science-data-editors.github.io/guidance/" target="_blank">Data and Code Guidance by Data Editors</a> developed by Lars Vilhuber and colleagues, which is used by the <a href="https://www.aeaweb.org/journals" target="_blank"><em>American Economic Association</em> journals</a>, <a href="https://onlinelibrary.wiley.com/journal/15405982" target="_blank"><em>Canadian Journal of Economics</em></a>, the <a href="https://academic.oup.com/restud" target="_blank"><em>Review of Economic Studies</em></a>, and the <a href="https://academic.oup.com/ej" target="_blank"><em>Economic Journal</em></a> as a reference.</p>
 				<p>Several journals in political science also require pre-publication verification. See for example <a href="https://www.cambridge.org/core/journals/state-politics-and-policy-quarterly/information/instructions-contributors" target="_blank"><em>State Politics & Policy Quarterly</em></a>, <a href="https://www.cambridge.org/core/journals/political-analysis/information/instructions-contributors" target="_blank"><em>Political Analysis</em></a>, and the <a href="https://ajps.org/" target="_blank"><em>American Journal of Political Science</em></a>.</p>
+				
+				<h4 id="resources-open-research-plan-doc">Resources for open research planning & documentation</h4>
+				<p>Many platforms can aid researchers in making their entire workflows open and transparent:</p>
```

Comment:
maybe preface by "for example" or so, since the list is obviously not comprehensive?